docs/installation-and-operations/configuration/README.md

```diff
@@ -160,9 +160,100 @@ Configuring OpenProject through environment variables is described in detail [in
 
 
 
-## Examples for common use cases
+# Seeding through environment
+
+OpenProject allows some resources to be seeded/created initially through configuration variables.
+
+| Topic                                                       | Description                                                  |
+| ----------------------------------------------------------- | ------------------------------------------------------------ |
+| [Initial admin user creation](#initial-admin-user-creation) | Changing attributes or passwords of the initially created administrator |
+| [Seeding LDAP connections](#seeding-ldap-connections)       | How to create an LDAP connection through configuration       |
+
+
+
+## Initial admin user creation
+
+**Note:** These variables are only applicable during the first initial setup of your OpenProject setup. Changing or setting them later will have no effect, as the admin user will already have been created.
+
+By default, an admin user will be created with the login and password set to `admin`. You will be required to change this password on first login.
+
+In case of automated deployments, you might find it useful to seed an admin user with password and attributes of your choosing. For that, you can use the following set of variables:
+
+
+
+```bash
+OPENPROJECT_SEED_ADMIN_USER_PASSWORD="admin" # Password to set for the admin user
+OPENPROJECT_SEED_ADMIN_USER_PASSWORD_RESET="true" # Whether to force a password reset on first login (true/false)
+OPENPROJECT_SEED_ADMIN_USER_NAME="OpenProject Admin" # Name to assign to that user (First and lastnames will be split on the space character)
+OPENPROJECT_SEED_ADMIN_USER_MAIL="admin@example.net" # Email attribute to assign to that user. Note that in packaged installations, a wizard step will assign this variable as well.
+```
+
+
+
+## Seeding LDAP connections
+
+OpenProject allows you to create and maintain an LDAP connection with optional synchronized group filters. This is relevant for e.g., automated deployments, where you want to trigger the synchronization right at the start.
```

Comment:
Explicitly stating that the seeding can be triggered multiple times and that this will add and alter but not remove ldap connections would clarify that part.